src/CMakeLists.txt

```diff
@@ -185,22 +185,24 @@ target_link_libraries (${BIN_FUSE} ${FUSE_LIB} ${PROJECT_NAME})
 set_target_properties (${BIN_FUSE} PROPERTIES COMPILE_DEFINITIONS FUSE_USE_VERSION=26)
 set_target_properties (${BIN_FUSE} PROPERTIES LINK_FLAGS "-pie -fPIE")
 add_custom_command (TARGET ${BIN_FUSE} POST_BUILD
-	COMMAND gzip -c ${DIS_MAN}/${BIN_FUSE}.1 > ${DIS_MAN}/${BIN_FUSE}.1.gz
+	COMMAND mkdir -p ${CMAKE_CURRENT_BINARY_DIR}/${DIS_MAN}/
```

Comment:
Option '-p' is supported by [FreeBSD](https://www.freebsd.org/cgi/man.cgi?query=mkdir&sektion=1) and [OSX](https://developer.apple.com/library/mac/documentation/Darwin/Reference/ManPages/man1/mkdir.1.html).